### DISCUSSION TITLE
Move links to certain crates

### DISCUSSION THREAD

**Discussion by hilmanski** [WebCrateApp/webcrate#25] [Q&A] [answered]
First, thank you for this tool. 
I've been using getpocket before, this is more likely what I want. 
I almost built something like this! luckily someone make it. So thank you for that 

Can we move a link to certain crate? 
Maybe start from the extension, we can choose on which crate we want to add this link
and later.. people can move the links between crate

Awesome work here!

**Comment by BetaHuhn** [comment, marked as answer]
Hey Hilman, thanks for the kinds words and for taking the time to create this discussion!

You can already move links between crates by simply dragging and dropping them! It's a bit hidden, but once you get the hang of it, it feels very natural.

As for the extension, that's definitely on my list! In the future you will be able to select a crate in the popup and it will directly add the link to it from the extension!

**Comment by hilmanski** [comment]
Its funny, I've just done it, and your notification shows up!
yes.. I didn't realize it.

Maybe(idea) show text for now as a hint for new user (you can drag and drop link to certain folder).

**Reply by BetaHuhn** [reply]
Haha nice! 

Good suggestion, thanks!

**Comment by BetaHuhn** [comment]
In the [latest version](https://github.com/WebCrateApp/browser-extension/releases/tag/v1.0.0) of the browser extension you can now select a crate to add the link to:
 
![image](https://user-images.githubusercontent.com/51766171/131148976-62b02913-d280-47cf-a068-41ca9a19e7d2.png)

It's already up on [Firefox Add-On Store](https://addons.mozilla.org/en-US/firefox/addon/webcrate/) and should be going live on the [Chrome WebStore](https://chrome.google.com/webstore/detail/webcrate-browser-extensio/hfioihmgjcjdepficckgginjciabkndc) soon.
